pueblo/ngr/core.py

```diff
@@ -44,14 +48,20 @@ def __init__(self, thing: t.Any, options: t.Dict) -> None:
         self.runner: t.Optional[RunnerBase] = None
         self.type: t.Optional[ItemType] = None
         self.runners = {
+            # Language runners, listed alphabetically.
             ItemType.DOTNET: DotNetRunner,
+            ItemType.ELIXIR: ElixirRunner,
+            ItemType.GOLANG: GolangRunner,
+            ItemType.HASKELL: HaskellRunner,
             ItemType.JAVA: JavaRunner,
             ItemType.JAVASCRIPT: JavaScriptRunner,
-            ItemType.MAKE: MakeRunner,
+            ItemType.JULIA: JuliaRunner,
```

Comment:
We will run those conditionally only, and probably separate the test suite.
```
=========================== short test summary info ============================
FAILED tests/test_ngr.py::test_ngr_sample[ngr/python-poetry] - FileNotFoundError: [Errno 2] No such file or directory: 'poetry'
FAILED tests/test_ngr.py::test_ngr_sample[ngr/elixir] - FileNotFoundError: [Errno 2] No such file or directory: 'mix'
FAILED tests/test_ngr.py::test_ngr_sample[ngr/haskell] - subprocess.CalledProcessError: Command '['stack', 'build']' returned non-zero exit status 1.
FAILED tests/test_ngr.py::test_ngr_sample[ngr/golang] - subprocess.CalledProcessError: Command '['go', 'test', '-v']' returned non-zero exit status 
```

Reply:
- The test suite has been separated, there is a dedicated [Tests for pueblo.ngr](https://github.com/pyveci/pueblo/actions/workflows/ngr.yml) workflow now.
- All dependencies have been satisified, so all newly added project samples for the enumerated languages build successfully now.